<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bread</class>
 <widget class="QWidget" name="bread">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>1380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>1380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1041</width>
     <height>1521</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/bread_page/bread_page_back.png);</string>
   </property>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/button/go_back_btn.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>1230</y>
      <width>229</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>229</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/button/Forward Arrow.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>322</width>
      <height>322</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>322</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bread_page/click_off/bread_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>630</y>
      <width>322</width>
      <height>322</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>322</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bread_page/click_off/bread_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>322</width>
      <height>322</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>322</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bread_page/click_off/bread_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>630</y>
      <width>322</width>
      <height>322</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>322</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bread_page/click_off/bread_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>322</width>
      <height>322</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>322</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bread_page/click_off/bread_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>322</width>
      <height>322</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>322</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/bread_page/click_off/bread_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
